screens/auth/SignUpScreen.js

```diff
@@ -228,39 +264,135 @@ export default class SignUp extends React.Component {
-          placeholder="Phone Number"
-          onChangeText={text => this.setState({ phoneNumber: text })}
-          value={this.state.phoneNumber}
-          keyboardType="number-pad"
-          maxLength={10}
-        />
-        <Input
-          placeholder="Password"
-          secureTextEntry
-          onChangeText={text => this.setState({ password: text })}
-          value={this.state.password}
-        />
-        <Button title="Sign Up" onPress={() => this.handleSubmit()} />
-        <Button
-          title="Already have an account? Log in"
-          onPress={() => this.props.navigation.navigate('Login')}
-        />
-        <Button title="Testing Bypass" onPress={() => this._devBypass()} />
-      </SignUpContainer>
+      <ScrollView showsVerticalScrollIndicator={false}>
+        <AuthScreenContainer>
+          <BigTitle>Sign Up</BigTitle>
+          <FormContainer>
+            <TextFieldContainer>
+              <Caption color={this.state.indicators[signUpFields.NAME]}>
+                Name
+              </Caption>
+              <TextField
+                onBlur={() => this.onBlur(signUpFields.NAME)}
+                onFocus={() => this.onFocus(signUpFields.NAME)}
+                placeholder="Name"
+                onChangeText={text => this.setState({ firstName: text })}
+                value={this.state.firstName}
+                borderColor={this.state.indicators[signUpFields.NAME]}
+              />
+              <InputNoticeContainer>
+                <Caption color={Colors.secondaryText}>
+                  Note: this is how clerks will greet you!
+                </Caption>
+              </InputNoticeContainer>
+            </TextFieldContainer>
+            <TextFieldContainer>
+              <TextField
+                onBlur={() => this.onBlur(signUpFields.NAME)}
+                onFocus={() => this.onFocus(signUpFields.NAME)}
+                placeholder="Last Name"
+                onChangeText={text => this.setState({ lastName: text })}
+                value={this.state.lastName}
+                borderColor={this.state.indicators[signUpFields.NAME]}
+              />
+            </TextFieldContainer>
+
+            <TextFieldContainer>
+              <Caption color={this.state.indicators[signUpFields.PHONENUM]}>
+                Phone Number
+              </Caption>
+              <TextField
+                onBlur={() => this.onBlur(signUpFields.PHONENUM)}
+                onFocus={() => this.onFocus(signUpFields.PHONENUM)}
+                placeholder="Phone Number"
+                onChangeText={text => this.setState({ phoneNumber: text })}
+                value={this.state.phoneNumber}
+                keyboardType="number-pad"
+                maxLength={10}
+                borderColor={this.state.indicators[signUpFields.PHONENUM]}
+              />
+              <InputNoticeContainer>
+                <Caption color={Colors.secondaryText}>
+                  Must be a valid phone number
```

Comment:
This should only pop up if the phone number entered is not valid, and should go away once it is valid. Also should be the dark orange color